extensions/amp-list/0.1/amp-list.js

```diff
@@ -892,14 +898,18 @@ export class AmpList extends AMP.BaseElement {
             this.unlistenLoadMore_ = null;
           }
         }).catch(() => {
-          this.mutateElement(() => {
-            this.loadMoreService_.setLoadMoreFailed();
-          });
-          const loadMoreFailedClickable = this.loadMoreService_
-              .getLoadMoreFailedClickable();
-          this.unlistenLoadMore_ = listen(
-              loadMoreFailedClickable,
-              'click', () => this.loadMoreCallback_(/*opt_reload*/ true));
+          this.mutateElement(() => this.loadMoreService_.setLoadMoreFailed())
+              .then(() => {
+                this.attemptToFitLoadMoreElement_(
```

Comment:
There are two `then()` blocks here. The first block should either return a `Promise` if the second block needs to wait for something to resolve, or the two blocks should be combined.

Reply:
Yes. Great catch. Thank you!

Reply:
Fixed this.